trace-utils/src/trace_utils.rs

```diff
@@ -594,9 +596,32 @@ pub fn collect_trace_chunks<T: tracer_payload::TraceChunkProcessor>(
     tracer_header_tags: &TracerHeaderTags,
     process_chunk: &mut T,
     is_agentless: bool,
-) -> TracerPayloadCollection {
+    deduplicate_strings: bool,
```

Comment:
Any reason why the input param `deduplicate_strings` shouldn't more directly convey the intent of using v05? The current name is ambiguous, imo.